proposals/000-index.txt

```diff
@@ -259,6 +260,7 @@ Proposals by status:
    295  Using ADL for relay cryptography (solving the crypto-tagging attack)
    296  Have Directory Authorities expose raw bandwidth list files
    299  Preferring IPv4 or IPv6 based on IP Version Failure Count
+   306  A Tor Implementation of IPv6 Happy Eyeballs
```

Comment:
Please let the merger modify the index.
If each pull request modifies the index, we get a lot of conflicts.